walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -79,15 +94,15 @@
    */
   public Collection<TimeRange> queryHelper(Collection<Event> separateEvents, MeetingRequest request) {
     // Map each {@code Event} to it's {@code TimeRange}.
-    List<TimeRange> TimeRangeList = separateEvents.stream()
-                     .map(a -> a.getWhen())
+    List<TimeRange> timeRangeList = separateEvents.stream()
+                     .map(Event::getWhen)
                      .collect(Collectors.toList());
 
     // Resulting Collection
     Collection<TimeRange> openTimeRanges = new ArrayList();
 
     int currentMinTime = 0;
-    for(TimeRange tr : TimeRangeList) {
+    for(TimeRange tr : timeRangeList) {
```

Comment:
nit: Forgot to mention earlier, but style guide is to have a space after reserved words like if/for

`for (` and `if (`

Comment is for here and elsewhere in the file 

See https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Reply:
Done.